src/string_pool.cr

```diff
@@ -35,14 +68,30 @@ class StringPool
     entry
   end
 
+  # Calls get(str : UInt8*, len) with the specified memoryIO. 
```

Comment:
Same as above (and also for the next method), say that it returns the string for the contents of the MemoryIO/String, without saying it just ends up invoking another method (otherwise the user has to jump between methods to understand how they work).